### PR TITLE
Style/pre commit hooks

### PR DESCRIPTION
**Pull request by lswank**
This should contain zero breaking changes and should just be able to be merged into the mainline. This is _only_ adding pre-commit styling hooks / Black and letting Black do its thing with whitespace. (It did one ugly thing with the comments, but we can always go back in and fix that if we need to.)

### CODE REVIEW

File: adjust [hunk 1]
```diff
@@ -15,36 +15,25 @@ VERSION = "0.0.1"
...
-            'max': 16, # GiB
-            'step': 0.125 # GiB
-        },
-        'cpu': {
-            'min': 0.125, # cores
-            'max': 4, # cores
-            'step': 0.125 # cores
-        }
+    "k8s": {
+        "mem": {"min": 0.125, "max": 16, "step": 0.125},  # GiB  # GiB  # GiB
```
Comment (lswank) [inline adjust:19]: @rstarmer  This looks really ugly, but apparently this is what Black does to these comments. I'm totally open to different Black configs, but I think Black is not very configurable by design from what I briefly read.

File: adjust [hunk 2]
```diff
@@ -206,371 +196,684 @@ class HarnessDriver(Adjust):
...
-            assert isinstance(v.get('max', defaults[k]['max']), (int, float)), 'range setting {} config was malformed; max must be a number. Found {}.'.format(k, v.get('max'))
-            assert isinstance(v.get('step', defaults[k]['step']), (int, float)), 'range setting {} config was malformed; step must be a number. Found {}.'.format(k, v.get('step'))
-            # TODO: require default for settings?
-            assert isinstance(v.get('default', 0), (int, float)), 'range setting {} config was malformed; default must be a number. Found {}.'.format(k, v.get('default'))
-                
-            assert v.get('min', defaults[k]['min']) <= v.get('max', defaults[k]['max']), \
-                'range setting {} config was malformed; supplied/default min ({}) is higher than supplied/default max ({})'\
-                    ''.format(k, v.get('min', defaults[k]['min']), v.get('max', defaults[k]['max']))
+        assert h.get(
+            "opsani_account"
```
Comment (lswank) [inline adjust:212]: @rstarmer Black also seems very arbitrary in what it splits into multi-line and what it wants to group together. 🤷‍♂️

File: adjust [hunk 3]
```diff
@@ -206,371 +196,684 @@ class HarnessDriver(Adjust):
...
+                        v.get("min", defaults[k]["min"]),
+                        v.get("max", defaults[k]["max"]),
+                        v.get("step", defaults[k]["step"]),
+                    )
+                )
 
         self.config = h
 
-        with open('/etc/harness-auth/token', 'r') as auth_file:
+        with open("/etc/harness-auth/token", "r") as auth_file:
```
Comment (lswank) [inline adjust:352]: @rstarmer Apparently Black is anti-single-quote

**Review by rstarmer — APPROVED** [review]
LGTM

File: adjust (hunk 1 above)
Comment (rstarmer) [reply adjust:19]: Yes, black isn't particularly configurable. Make it all consistent, so Black is the answer...

**Pull request merged**